### DISCUSSION TITLE
Notes - Automne 2022

### DISCUSSION THREAD

**Discussion by Juknum** [ae-utbm/.github#5] [Notes]
## Remise en état de l'infrastructure

- Réinstallation de [Sentry](https://sentry.io/) sur la machine fournie par la DSI (disponible sur <https://ae2.utbm.fr>)
- Installation de [Grafana](https://grafana.com/), outil de monitoring utilisé pour surveiller l'état de l'infrastructure : <https://ae-dev.utbm.fr/grafana>
- Migration du Bitwarden vers sa solution open-source & auto-hébergée : <https://vault.ae.utbm.fr>
- Fermeture du GitLab en faveur de GitHub.
- Mise en place de Crowdin pour les traductions des applications (notamment le site 4) : <https://ae-utbm.crowdin.com/u/>
- Mise à jour d'Ubuntu

## Développement du Site 4

- Premiers tests d'implémentations (cf. [Site 4](https://github.com/ae-utbm/sith4/tree/project-core) & [API](https://github.com/ae-utbm/api/tree/project-core)).
- Choix d'une palette de couleurs pour le site ([Figma](https://www.figma.com/file/Lc5YU1E5XI0XEvG4w2eEp4/Sith-V4-(WIP)))
- Grosse avancée des travaux sur les composants de base du site.

## Amélioration continue du Site 3

- Remise en état du site de test : <https://ae-taiste.utbm.fr>
- Remise en place de Sentry lorsqu'une erreur survient sur le site
- Mise en place de 3DSecure v2 dans l'Eboutic
- [Refonte](https://github.com/ae-utbm/sith3/pull/477) & [améliorations](https://github.com/ae-utbm/sith3/pull/522) de l'interface utilisateur sur l'Eboutic
- Ajout du logo de la promo 23 👀
- Plein de fixes et de petites améliorations

**Comment by imperosol** [comment]
Pour l'eboutic, on est plus sur une refonte quasi-intégrale que sur juste une amélioration de l'interface.

**Reply by Juknum** [reply]
Le refonte passe dans la mise en place du 3DSecure ? 🤔

**Reply by imperosol** [reply]
Non, elle intervient déjà bien avant. Tu peux revérifier [cette PR](https://github.com/ae-utbm/sith3/pull/477). La totalité du front de la boutique en ligne a été changée, ainsi que la quasi-totalité des vues (et ce qui n'a pas été édité à ce moment-là l'a été pour le 3DSv2). Il ne doit pas y avoir plus d'une trentaine de lignes qui n'ont pas été modifiées ou ajoutées ce semestre dans `eboutic/views.py`, sur les 244 du total.
